auto_nag/scripts/spambug.py

```diff
@@ -7,10 +7,20 @@
 from auto_nag.bzcleaner import BzCleaner
 from auto_nag.utils import nice_round
 
+COMMENT = """
+The [Bugbug](https://github.com/mozilla/bugbug/) bot thinks this bug is invalid, with a confidence of {}%.
+Please revert this change in case of error.
```

Comment:
Should we say something like: "If you think the bot is wrong, please reopen the bug"?
It could be easier to understand for new bug filers